src/jbuild

```diff
@@ -63,8 +63,8 @@
     (progn
      (with-stdout-to popcnt_test.c
       (echo "int main(int argc, char ** argv) { return __builtin_popcount(argc); }"))
-     (system "${CC} -mpopcnt -c popcnt_test.c 2> ${null} && \
-echo '(-mpopcnt)' > ${@} || echo '()' > ${@}"))))))
+     (with-stdout-to ${@} (progn (echo "(") (system "${CC} -mpopcnt -c popcnt_test.c 2> ${null} && \
+echo -mpopcnt || true 2> ${null} || ver > ${null}") (echo ")"))))))))
```

Comment:
cmd doesn't have an equivalent to `true` so `ver` is used instead (it display the OS version) as it also returns a zero exit code. The problem is that `false 2>/dev/null || echo foo>/dev/null` return `0` on Unix but `false 2>nul || echo foo>nul` returns `1` on Windows (because echo is a built-in which doesn't change the exit code of the pipe)